libraries/fc/src/log/console_appender.cpp

```diff
@@ -105,6 +105,7 @@ namespace fc {
 
       std::string line;
       line.reserve( 256 );
+      line += "[ ";
```

Comment:
I'm not sure that we want this, it changes the existing output format.

Reply:
The reason I added it is because there is an unmatched "]" at the end of the line. If this output is verified in some tests, we should remove it. However, I did not see any failure from buildkite. I can remove it if you think it is risky for some automation runs.

Reply:
It is probably best to stay consistent with what was previously there, in the case that logs are being parsed and used.

Reply:
Reverted that line.